recipes/ca-certificates/meta.yaml

```diff
@@ -0,0 +1,43 @@
+{% set version = "2016.2.28" %}
+
+package:
+  name: ca-certificates
+  version: {{ version }}
+
+# This does not have a source because it pulls these from certifi at present.
+
+build:
+  number: 0
+  skip: true  # [win or not py35]
+  # As openssl includes these (dependency of Python), they will be removed afterwards.
+  # This is our way of ensuring these files are kept anyways. Once conda-forge has its
+  # own openssl, this will no longer be necessary.
+  always_include_files:
+    - ssl/cert.pem
+    - ssl/cacert.pem
```

Comment:
We place these in the same location as `openssl` does, but ours come from `certifi`. However, `conda-build` thinks these had come from `openssl`. So, we have to explicitly tell `conda-build` to keep them. This won't be an issue once we have our own `openssl`. ( https://github.com/conda-forge/staged-recipes/pull/553 )